plugins/eslint-plugin-aws-local/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "eslint-plugin-aws-local",
+    "version": "1.0.0",
+    "description": "Local custom lint rules for AWS Toolkit VSCode",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "tsc",
+        "test": "npm run build && mocha dist/test --recursive"
+    },
+    "devDependencies": {
+        "eslint": "^8.26.0",
+        "mocha": "^10.1.0",
+        "typescript": "^5.0.4"
```

Comment:
This (and the tsconfig) is the main downside of a subproject, I guess. We need to maintain its dependencies.